store/views.py

```diff
@@ -375,31 +383,127 @@ def checkout(request):
     return render(request, "order_complete.html")
 
 
-# add a new book for author
+# add a new book for seller
 class AddBook(TemplateView):
```

Comment:
Why did you used here template view if you want to override get and post method? please explain

Reply:
Noted, i would have used view class here as i am overriding the get and post methods

Reply:
@MansiS117 can you please do some research if you can use Template view and also pass messages for specific cases.

Reply:
sir, i have made the suggested changes